response_operations_ui/templates/conversation-view.html

```diff
@@ -0,0 +1,51 @@
+{% extends "layouts/base.html" %}
+
+{% block page_title %} Create Message | Survey Data Collection {% endblock %}
+
+{% block main %}
+
+
+
+<h1 class="saturn" name="page-messages-title">{{ subject }}</h1>
+
+<div class="secure-message-conversation block">
+    <div class="secure-message-sent-message-meta">
+         <div class="secure-message--meta venus"><span class="sm-form-meta-label">Survey: </span><span id="survey">{{ messages[0].get('survey') }}</span></div>
+         <div class="secure-message--meta venus"><span class="sm-form-meta-label">RU Ref: </span><span id="ruref">    {{ messages[0].get('ru_ref') }}</span></div>
+         <div class="secure-message--meta venus"><span class="sm-form-meta-label">Business: </span><span id="ru">{{ messages[0].get('business_name') }}</span></div>
+         <div class="secure-message--recipient venus"><span class="sm-form-recipient-label">To: </span><span id="recipient">    {{ messages[0].get('to') }}  </span></div>
+
+    </div>
+    {% for message in messages %}
+       <div class="secure-message-sent-message">
+    {% if message.get('internal') == True %}
+        <div id="smC1M1" class="secure-message-sent-message-meta sm-from-ons">
```

Comment:
What's the deal with these ids?  Isn't there a more human readable way to describe this field?

Reply:
@insacuri id's have been changed but they were originally ones the UX guys have used in their templates